src/components/Channel/Channel.tsx

```diff
@@ -219,17 +434,21 @@ const ChannelInner = ({
   );
 
   const loadMore = useCallback(
-    async (limit = 100) => {
-      if (!online.current || !window.navigator.onLine) return 0;
+    // eslint-disable-next-line @typescript-eslint/no-inferrable-types
```

Comment:
?

Reply:
some weirdness happening with the useCallback, it will infer the type as number but then later in the file it's referenced as any